integration/container/nat_test.go

```diff
@@ -56,6 +56,7 @@ func TestNetworkLocalhostTCPNat(t *testing.T) {
 }
 
 func TestNetworkLoopbackNat(t *testing.T) {
+	skip.If(t, testEnv.GitHubActions, "FIXME") // https://github.com/moby/moby/issues/41561
```

Comment:
Did we already have a `testEnv.GithubActions`? Isn't that the one added in the next commit? (if so, the order of commits needs to be changed).

Can you also include the URL as part of the `"FIXME"` string (as I suggested in my other comment?). It's a bit more noisy, but shows up in logs, hopefully pointing readers to "let's have a look at fixing" 😂)